test/collection_test.js

```diff
@@ -2265,4 +2278,88 @@ describe("Collection", () => {
       });
     });
   });
+
+  /** @test {Collection#execute} */
+  describe("#execute", () => {
+    it("should support get", () => {
+      const articles = testCollection();
+      return articles.create(article)
+        .then(result => {
+          const id = result.data.id;
+          return articles.execute(txn => txn.get(id), {preloadIds: [id]});
+        })
+        .then(result => expect(result.data.title).eql("foo"));
```

Comment:
Nit: please put the `.eql` call on its own line, I've found that most often this eases reading the tests a lot.